### PR TITLE
Updated REAME.md to use GitHub's syntax highlighting

### PR DESCRIPTION
**Pull request by neanias**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ re-hash those passwords. This vulnerability only affected the JRuby gem.
 
 ## How to install bcrypt
 
-    gem install bcrypt
+```ruby
```
Comment (tjschuck) [inline README.md]: This is not Ruby.  It's a CLI command.

Comment (neanias) [inline README.md]: Good point, I'll change it

File: README.md [hunk 2]
```diff
@@ -116,7 +128,9 @@ Because this process is not reversible, there's no way to go from the hash back
 
 In other words:
 
-    hash(p) #=> <unique gibberish>
+```ruby
```
Comment (tjschuck) [inline README.md]: Not specifically Ruby -- this is just illustrative pseudocode.

File: README.md [hunk 3]
```diff
@@ -127,13 +141,17 @@ You can store the hash and check it against a hash made of a potentially valid p
...
+PrecomputedPassword.find_by_hash(<unique gibberish>).password #=> "secret1"
+```
 
 ### Salts
 
 The solution to this is to add a small chunk of random data -- called a salt -- to the password before it's hashed:
 
-    hash(salt + p) #=> <really unique gibberish>
+```ruby
```
Comment (tjschuck) [inline README.md]: Pseudocode, not Ruby.

**Comment by tjschuck** [comment]
Done in #159 instead.

**Pull request closed without merging**